pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -31,16 +33,110 @@ import groovy.json.JsonSlurper
  *
  */
 
-def getJavaVersionNumber(version) {
-    // version should be something like "jdk8u"
-    def matcher = (version =~ /(\d+)/)
-    return Integer.parseInt(matcher[0][1])
+def addOr0(map, name, matched, groupName) {
```

Comment:
Purpose of this function?  As a standalone, it's not clear to me.  Perhaps a comment.